sky/provision/common.py

```diff
@@ -21,6 +21,10 @@ class ProvisionError(RuntimeError):
     errors: List[Dict[str, str]]
 
 
+class TeardownError(Exception):
```

Comment:
This may suggest to the reader as an error thrown during `sky down`. How about `StopFailoverError`?

Reply:
Good point! Renamed it. Thanks!